src/port.js

```diff
@@ -0,0 +1,17 @@
+function Port(name) {
+  this.port = name;
+  this.ships = [];
+}
+
+Port.prototype.addShip = function addShip(ship) {
+  this.ships.push(ship);
+};
+
+Port.prototype.removeShip = function removeShip(shipToRemove) {
+  this.ships.splice(this.ships.findIndex(ship => ship === shipToRemove), 1);
```

Comment:
Nice, but what if `removeShip` was called with a ship object which didn't exist in `this.ships`?